kotlintest-assertions/src/main/kotlin/io/kotlintest/matchers/StringMatchers.kt

```diff
@@ -23,8 +23,6 @@ fun startWith(prefix: String) = neverNullMatcher<String> { value ->
 }
 
 fun haveSubstring(substr: String) = include(substr)
-@Deprecated("use should include(substring)", ReplaceWith("include(substr)", "io.kotlintest.matchers.string.include"))
```

Comment:
How long has this one been deprecated for ?

Reply:
04/2018